account_product_move/models/product_template.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2022 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    journal_tmpl_id = fields.Many2one(
```

Comment:
If you make the relation between product.template and account.product.move a Many2many, you can have several moves in a product (for instance to support different currencies or journals), but you can also re-use the account.product.move for several products. For instance if they all have the same cost, or the same tax, or the same whatever moves.